docs/source/getting_started/handling_time.ipynb

```diff
@@ -243,15 +251,29 @@
     "Featuretools can automatically add last time indexes to every DataFrame in an ``Entityset`` by running ``EntitySet.add_last_time_indexes()``. When using a training window, if a `last_time_index has` been set, Featuretools will check to see if the `last_time_index` is after the start of the training window. That, combined with the cutoff time, allows DFS to discover which data is relevant for a given training window.\n",
     "\n",
     "\n",
-    "## Excluding data at cutoff times\n",
-    "\n",
+    "## Excluding data at cutoff times"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "e7953105",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
     "The ``cutoff_time`` is the last point in time where data can be used for feature\n",
     "calculation. If you don't want to use the data at the cutoff time in feature\n",
     "calculation, you can exclude that data by setting ``include_cutoff_time`` to\n",
-    "``False`` in [featuretools.dfs](../generated/featuretools.dfs.html#featuretools.dfs) or [featuretools.calculate_feature_matrix](../generated/featuretools.calculate_feature_matrix.html#featuretools.calculate_feature_matrix).\n",
+    "``False`` in :func:`featuretools.dfs` or:func:`featuretools.calculate_feature_matrix`.\n",
```

Comment:
The calculate feature matrix link didn't render right here for me.

![image](https://user-images.githubusercontent.com/4307001/129258977-b5cda1be-99bc-4ec7-82eb-07fd4e52cfa9.png)

Reply:
ah, it's that there's no space `or:func` - fixed!